python/cac_tripplanner/cac_tripplanner/settings.py

```diff
@@ -0,0 +1,96 @@
+"""
+Django settings for cac_tripplanner project.
```

Comment:
Seems like it would be fine (and possibly a bit of an improvement) after thinking about it a bit. Sure, I'm game for giving it a try.